test/multiple/example/src/vl-example-all.js

```diff
@@ -0,0 +1,16 @@
+import { define, awaitScript } from '/node_modules/vl-ui-core/dist/vl-core.js';
+
+import { VlExampleOne } from "/src/vl-example.js";
+import { VlExampleOther } from "/src/vl-example-other.js";
+
+Promise.all([
+    awaitScript('vl-map-openlayers', '/node_modules/vl-mapactions/lib/openlayers/dist/ol.js')
```

Comment:
Kunnen we hier niet beter verwijzing maken naar een voorbeeld script ipv openlayers?

Reply:
maakt niet zoveel uit hier omdat die code niet gaat uitgevoerd worden, maar ik wil dat wel aanpassen.